tools/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apt update \
        libleveldb-dev libleveldb1d
 
 FROM build AS install
-ARG ROCKSDB=9.2.1
+ARG ROCKSDB=9.2.3
```

Comment:
I think bumping the version of `ROCKSDB` is causing the tests to fail. Is it necessary?